### ISSUE TITLE
Clipboard is not working if "Run in the background" is off

### ISSUE THREAD

**Issue by arielnmz** [mijorus/smile#36]
To reproduce:

- Turn *Run in the background* off
- Click on an emoji
- Window closes
- Press Ctrl+V in an input field
- Nothing is pasted (also clipboard gets emptied)

```
$ flatpak info it.mijorus.smile

Smile - An emoji picker

          ID: it.mijorus.smile
         Ref: app/it.mijorus.smile/x86_64/stable
        Arch: x86_64
      Branch: stable
     Version: 2.0.9
     License: GPL-3.0-or-later
      Origin: flathub
  Collection: org.flathub.Stable
Installation: user
   Installed: 16.2 MB
     Runtime: org.gnome.Platform/x86_64/43
         Sdk: org.gnome.Sdk/x86_64/43

      Commit: 0703c1374c16f1a314d5d1cbcb9254cda0a1b2c38eebcf46cafb6173ac6a5a2f
      Parent: d4802662a5c0c672c5647d138ab791abce393e2569f54563629d60f45c3f86bb
     Subject: Update it.mijorus.smile.json (c3d93d5e)
        Date: 2023-02-12 18:20:00 +0000

$ cat /etc/fedora-release
Fedora release 35 (Thirty Five)

$ sudo dnf info xorg-x11-server-Xorg
Last metadata expiration check: 3:35:57 ago on Tue 21 Feb 2023 12:05:35 PM PST.
Installed Packages
Name         : xorg-x11-server-Xorg
Version      : 1.20.14
Release      : 9.fc35
Architecture : x86_64
Size         : 3.7 M
Source       : xorg-x11-server-1.20.14-9.fc35.src.rpm
Repository   : @System
From repo    : updates
Summary      : Xorg X server
URL          : http://www.x.org
License      : MIT
Description  : X.org X11 is an open source implementation of the X Window System.  It
             : provides the basic low level functionality which full fledged
             : graphical user interfaces (GUIs) such as GNOME and KDE are designed
             : upon.
```

**Comment by mijorus** [comment]
I'm sorry but I can't replicate it. I'm trying my best on all the machines I can find but it runs just fine on my fedora VM

**Comment by mijorus** [comment]
it's some sort of issue with GTK and its clipboard manager, it happened to me and other users in the past, but it was supposed to be fixed now.

Basically the process quits before the clipboard receives the new content, I have no idea how to debug an issue like this.

In the meanwhile, please just make sure that you are NOT using apps like Flatstat to manage or alter permissions

**Comment by mijorus** [comment]
Please check out this commit:
https://github.com/mijorus/smile/commit/c931a956a8909debc1bffe2486ed199c8e90c839

**Comment by mijorus** [comment]
version 2.9.3 will be using wl-clipboard as backend, which should (finally) fix this issue once for all

**Comment by mijorus** [comment]
Also, closes #76

**Comment by mijorus** [comment]
Thanks for contributing with this bug report.

If you want to support my work, please consider a small donation.

https://ko-fi.com/mijorus
